scripts/win32.bat

```diff
@@ -5,12 +5,27 @@ exit /b
 
 ----- Begin wsf script --->
 <job><script language="VBScript">
+
+Private OSDrive
+
 strComputer = "."
 
 Set objWMIService = GetObject("winmgmts:\\" & strComputer & "\root\cimv2")
 
 Set colDiskDrives = objWMIService.ExecQuery("SELECT * FROM Win32_DiskDrive")
 
+Set colOperatingSystems = objWMIService.ExecQuery ("SELECT SystemDrive FROM Win32_OperatingSystem")
+
+For Each objOperatingSystem in colOperatingSystems
+    On Error Resume Next
+    '' get the OS System Drive if exists
+    OSDrive = objOperatingSystem.Properties_("SystemDrive")   
+    If Err.Number <> 0 Then
+        OSDrive = Err.Description
+        Err.Clear
```

Comment:
Maybe worth calling `Err.Clear` *before* the loop starts as well, since if an error exists before the loop, `OSDrive` will not get the right value, and the drive will be reported as `system: false`.